pkg/mcp/source/client_source.go

```diff
@@ -51,6 +54,32 @@ func NewClient(client mcp.ResourceSinkClient, options *Options) *Client {
 
 var reconnectTestProbe = func() {}
 
+func (c *Client) sendDummyResponse(stream Stream) error {
```

Comment:
Nit: sendTriggerResponse? (with comments explaining why this might be needed?)

Reply:
Done